src/components/common/ChangeTypeButton.js

```diff
@@ -0,0 +1,38 @@
+import React, {useState} from 'react';
+
+function ChangeTypeButton({ TextCT, DItem1, DItem2, DItem3, type }) {
```

Comment:
This looks like it's very very similar to the previous one. Do we really need two components for it?